src/pages/StartPage.tsx

```diff
@@ -32,7 +32,12 @@ const StartPage = () => {
             Precinct {precinctId}
           </p>
           <p>
-            <LinkButton primary to="/instructions/">
+            <LinkButton
+              primary
+              to="/instructions/"
+              id="next"
+              aria-label="Select Next to Get Started."
```

Comment:
For consistency…

```suggestion
              aria-label="Press the right arrow to get started."
```

…or…

```suggestion
              aria-label="Press select to get started."
```